rpc/src/v1/impls/ibc.rs

```diff
@@ -102,28 +102,33 @@ where
     }
 
     fn compose_header(&self, block_number: u64) -> Result<Option<Bytes>> {
+        // FIXME: Currently compose_header() works only for at most bestblock-1
```

Comment:
Could you give more context?
Like "block n's seal is stored in the next block's header".

Reply:
Ok